### PR TITLE
add correct subtotal to order receipt page

### PR DESCRIPTION
**Pull request by john-raymon**
- solves https://github.com/chec/commercejs-nextjs-demo-store/issues/93#issuecomment-673066962

### CODE REVIEW

File: components/checkout/Confirm.js [hunk 1]
```diff
@@ -125,14 +125,14 @@ class Confirm extends Component {
...
-                        <p className="text-right font-weight-medium">${orderReceipt.order.total_with_tax.formatted_with_code}</p>
+                        <p className="text-right font-weight-medium">${orderReceipt.order.subtotal.formatted_with_code}</p>
                       </div>
                   </div>
                   <div className="d-flex justify-content-between align-items-center mb-2 pt-3">
                     <p className="font-size-title font-weight-semibold">
                       Order total
                     </p>
-                    <p className="text-right font-weight-semibold font-size-title">${orderReceipt.order.total.formatted_with_code}</p>
+                    <p className="text-right font-weight-semibold font-size-title">${orderReceipt.order.subtotal.formatted_with_code}</p>
```
Comment (robbieaverill) [inline components/checkout/Confirm.js:135]: I think this should be `total_with_tax` - is there a reason not to use it here?

Comment (john-raymon) [reply components/checkout/Confirm.js:135]: Yes, since only the `order.subtotal` object is reflecting the correct total right now.

image

Comment (john-raymon) [reply components/checkout/Confirm.js:135]: See, #93 (comment).

Comment (ScopeyNZ) [reply components/checkout/Confirm.js:135]: Is there a discount?

Comment (robbieaverill) [reply components/checkout/Confirm.js:135]: This doesn't make any sense 🤔

Comment (ScopeyNZ) [reply components/checkout/Confirm.js:135]: Can you dump the full order receipt in a Gist and link it here? If this is wrong we really need to get a fix out for the API.

Comment (john-raymon) [reply components/checkout/Confirm.js:135]: Yes, on it now.

Comment (john-raymon) [reply components/checkout/Confirm.js:135]: @ScopeyNZ https://gist.github.com/john-raymon/6f897e1783c1d0e0bd74160fd344d7ec

Comment (john-raymon) [reply components/checkout/Confirm.js:135]: Thanks for pointing out the discount @ScopeyNZ.  lol 🤦🏽‍♂️

**Pull request merged**